src/Domain/CoordinateMaps/TimeDependent/Translation.cpp

```diff
@@ -236,24 +235,6 @@ double Translation<Dim>::root_finder(const double target_radius,
   if (upper_bound <= 0.) {
     return 0.;
   }
-  double function_at_lower_bound = std::numeric_limits<double>::min();
-  double function_at_upper_bound = (*f_of_r_)(upper_bound);
-  // bracket function requires a function to return std::optionals
-  const auto radius_bracket_function =
-      [this, &target_radius, &squared_function_of_time,
-       &root_factor](const double x) -> std::optional<double> {
-    return square(target_radius) +
-           ((*f_of_r_)(x)) *
-               (root_factor + squared_function_of_time * (*f_of_r_)(x)) -
-           square(x);
-  };
-  if (radius_bracket_function(lower_bound).value() <=
-      std::numeric_limits<double>::min()) {
-    return lower_bound;
```

Comment:
I haven't tested this being taken out, but does this still cover the cases where the root could be or is close to 0?

Reply:
It's hard to analyze this code with so many user-specified functions (`f_of_r_` and multiple functions of time).  Can you give an example of the sort of case you're worried about?

I'm also not entirely sure what `root` represents.  It appears to be some sort of radius, but I don't know of what.

Reply:
The `root` here represents the radius for the radial math function that will take the `target_coords` back to the `source_coords`. So if you have a point that gets translated in a way that looks very close to `gsl::at(result, i) += gsl::at(function_of_time_, i) * 1` (like the case when there's no radial falloff) then the root has to be 0. I think this can happen when the radius to your original coordinate is close to the origin and hasn't been translated very much yet.

Reply:
I see.  Not sure if there might have been problems there, so I rewrote the root-find so that the function is manifestly non-positive at r=0.